app/services/claims/submit.rb

```diff
@@ -1,18 +1,34 @@
 class Claims::Submit
   include ServicePattern
 
-  def initialize(claim:, claim_params:)
+  def initialize(claim:, claim_params:, user:)
     @claim = claim
     @claim_params = claim_params
+    @user = user
   end
 
   def call
     updated_claim.save!
+
+    # We will be implementing a different email for support users in another ticket
+    if user.type != "Claims::SupportUser" && status_changed_to_submitted
+      send_claim_submitted_notification_email
+    end
   end
 
   private
 
-  attr_reader :claim, :claim_params
+  attr_reader :claim, :claim_params, :user
+
+  def send_claim_submitted_notification_email
+    UserMailer.with(service: user.service).claim_submitted_notification(user, claim).deliver_later
+  end
+
+  def status_changed_to_submitted
+    return false if updated_claim.saved_change_to_status.nil?
+
+    true if updated_claim.saved_change_to_status[1] == "submitted"
```

Comment:
Unnecessary `true`, when `updated_claim.saved_change_to_status[1] == "submitted"` will return a boolean

Reply:
It might be more elegant to write this logic as:

```ruby
  def status_changed_to_submitted?
    updated_claim.saved_change_to_status? && updated_claim.submitted?
  end
```

Note that the method name ends with a `?` since this is technically a [predicate method](https://pragmati.st/2012/03/24/the-elements-of-ruby-style-predicate-methods/).